docs/how-to-build-with-ddn/with-trino.mdx

```diff
@@ -0,0 +1,465 @@
+touch app/connector/my_trino/compose.trino.yaml
+```
+
+```yaml title="Then, open the file and add the following:"
+services:
+  postgres:
+    image: postgres:15
+    container_name: postgres
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydb
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U myuser"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  trino:
+    image: trinodb/trino:latest
+    container_name: trino
+    depends_on:
+      postgres:
+        condition: service_healthy
+    ports:
+      - "8080:8080"
+    command: |
+      /bin/sh -c "
+      mkdir -p /etc/trino/catalog &&
+      echo 'connector.name=postgresql' > /etc/trino/catalog/postgres.properties &&
+      echo 'connection-url=jdbc:postgresql://postgres:5432/mydb' >> /etc/trino/catalog/postgres.properties &&
+      echo 'connection-user=myuser' >> /etc/trino/catalog/postgres.properties &&
+      echo 'connection-password=mypassword' >> /etc/trino/catalog/postgres.properties &&
+      /usr/lib/trino/bin/run-trino
+      "
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/v1/info"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+volumes:
+  pgdata:
+```
+
+```sh title="Run the container:"
+docker compose -f app/connector/my_trino/compose.trino.yaml up -d
+```
+
+The Trino container will start; you can access the UI — which will give you general information about your clusters —
+here: [`http://localhost:8080/`](http://localhost:8080/).
+
+### Step 5. Create a table in your PostgreSQL database
+
+```sh title="Enter the PostgreSQL container:"
+docker exec -it postgres psql -U myuser -d mydb
+```
+
+```sql title="Then, create your first table and seed the database:"
+--- Create the table
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  age INT NOT NULL
+);
+
+--- Insert some data
+INSERT INTO users (name, age) VALUES ('Alice', 25);
+INSERT INTO users (name, age) VALUES ('Bob', 30);
+INSERT INTO users (name, age) VALUES ('Charlie', 35);
+```
+
+You can verify this worked by using Adminer to query all records from the `users` table:
+
+```sql
+SELECT * FROM users;
+```
+
+### Step 6. Introspect your Trino instance
+
+```sh title="Next, use the CLI to introspect your Trino database:"
+ddn connector introspect my_trino
+```
+
+After running this, you should see a representation of your database's schema in the
+`app/connector/my_trino/configuration.json` file; you can view this using `cat` or open the file in your editor.
+
+```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
+ddn connector show-resources my_trino
+```
+
+### Step 7. Add your model
+
+```sh title="Now, track the table from Trino server as a model in your DDN metadata:"
+ddn model add my_trino "users"
```

Comment:
Other tuts we're not quoting these. Does it make a difference.

```suggestion
ddn model add my_trino users
```

Reply:
Good point! During one of the guides, I was using `"*"` for all models and I think I just got back in the habit.